src/ucs/sys/init.c

```diff
@@ -78,6 +78,7 @@ static void UCS_F_CTOR ucs_init()
     ucs_log_early_init(); /* Must be called before all others */
     ucs_global_opts_init();
     ucs_log_init();
+    ucs_cpu_init();
```

Comment:
should be done before log_init, and after global_opts_init

Reply:
initialization shifted